charts/metering-ansible-operator/templates/crds/report.crd.yaml

```diff
@@ -35,3 +35,154 @@ spec:
   - name: Age
     type: date
     JSONPath: .metadata.creationTimestamp
+  validation:
+    openAPIV3Schema:
+      type: object
+      required:
+      - spec
+      properties:
+        # note: see https://github.com/operator-framework/operator-metering/blob/master/Documentation/reports.md for more information
+        # note: apiVersion, kind, metadata is implicitly validated by API server
+        # note: can't use $ref so we wouldn't be able to check reportingStart < reportingEnd
```

Comment:
That actually isn't what `$ref` would allow you to do, and this kind of logic isn't possible that I'm aware of, but isn't worth calling out as a comment I don't think.